xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/OkexExchangeV3.java

```diff
@@ -10,6 +11,9 @@
 
 public class OkexExchangeV3 extends BaseExchange {
 
+  public static SimpleDateFormat timestampFormatter =
+      new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
DTO object should NOT have a dependancy to OkexExchangeV3 remove this from here

this is a standard ISO 8601 date time format - just use Date in DTOa, Jackson will handle it